refactor: log relative paths for file

algomaster99 opened the pull request:
Continuation of #1029

algomaster99 commented:
Checked for `/home/macaron` [here](https://github.com/oracle/macaron/actions/runs/13936281341/job/39005499709). Except for https://github.com/oracle/macaron/actions/runs/13936281341/job/39005499709#step:8:1825, I refactored every logging statement. Maybe that statement has been removed in future versions.

algomaster99 commented:
Once the CI checks are enabled, I can verify if I have taken care of all logging statments.

algomaster99 commented:
The typing annotations check does not fail for me locally. And this seems to be part of the code that I did not change so is there some issue with the CI? I have an older version of Python the build `3.11.0` vs [`3.11.11` in build](https://github.com/oracle/macaron/actions/runs/14147498459/job/39651904846?pr=1032#step:3:14).
```
Check typing annotations.................................................Failed
- hook id: mypy
- exit code: 1

src/macaron/database/views.py:147:59: error: Argument "callable_" to "execute_if" of "ExecutableDDLElement" has incompatible type "Callable[[BaseDDLElement, SchemaItem, Connection | None, list[Table] | None, Any | None, KwArg(Any)], bool]"; expected "DDLIfCallable | None"  [arg-type]
src/macaron/database/views.py:151:90: error: Argument "callable_" to "execute_if" of "ExecutableDDLElement" has incompatible type "Callable[[BaseDDLElement, SchemaItem, Connection | None, list[Table] | None, Any | None, KwArg(Any)], bool]"; expected "DDLIfCallable | None"  [arg-type]
```

behnazh-w commented:
> The typing annotations check does not fail for me locally. And this seems to be part of the code that I did not change so is there some issue with the CI? 

Yes. This [PR](https://github.com/oracle/macaron/pull/1035) fixes this typing error. Please rebase your branch on `staging` once the PR is merged.

algomaster99 commented:
@behnazh-w I rebased and force-pushed. My unit tests were failing because of some coverage requirement so I was not able to push, but for now I disable the `pre-push` hook. I don't think so it should affect this PR.

See error log:
```

FAIL Required test coverage of 60.0% not reached. Total coverage: 32.57%
=========================== short test summary info ============================
ERROR src/macaron/slsa_analyzer/analyzer.py
ERROR src/macaron/slsa_analyzer/checks/detect_malicious_metadata_check.py
ERROR tests/slsa_analyzer/checks/test_detect_malicious_metadata_check.py
ERROR tests/slsa_analyzer/checks/test_detect_malicious_metadata_check.py
ERROR tests/slsa_analyzer/test_analyzer.py
ERROR tests/slsa_analyzer/test_analyzer.py
ERROR tests/test_main.py
ERROR tests/test_main.py
!!!!!!!!!!!!!!!!!!! Interrupted: 8 errors during collection !!!!!!!!!!!!!!!!!!!!
============================== 8 errors in 4.53s ===============================
```

behnazh-w commented:
> @behnazh-w I rebased and force-pushed. My unit tests were failing because of some coverage requirement so I was not able to push, but for now I disable the `pre-push` hook. I don't think so it should affect this PR.

That's interesting. The unit tests seem to be passing [on the CI](https://github.com/oracle/macaron/pull/1032#issuecomment-2767662585), so there should be an issue with your local setup.

tromai commented:
Hi @algomaster99 It seems like there are more error messages for this issues. Could you please also include here any other error messages from pytest (if available) ?

algomaster99 commented:
>  so there should be an issue with your local setup.

It was complaining about my outdated TOML file. I simply ran `make upgrade` and it is now fixed.

> It seems like there are more error messages for this issues.

Hopefully, I fixed it my new commit [`1ce3676` (#1032)](https://github.com/oracle/macaron/pull/1032/commits/1ce3676e435b27011dac1479fb2b8d02bac645b8). (the message will be squashed anyway, right?)

behnazh-w commented:
> Hopefully, I fixed it my new commit [`1ce3676` (#1032)](https://github.com/oracle/macaron/pull/1032/commits/1ce3676e435b27011dac1479fb2b8d02bac645b8). (the message will be squashed anyway, right?)

Thanks! Yes, we will squash the commits.

behnazh-w commented:
@algomaster99 Thanks for the changes and we are almost ready to merge the PR. The commits however miss the `Signed-off-by: Your Name <you@example.org>` as instructed [here](https://github.com/oracle/macaron/blob/main/CONTRIBUTING.md#contributing-code), which can be done by the following `git commit` arg:
``` bash
git commit --signoff
```

algomaster99 commented:
@behnazh-w signed-off my commits. I overlooked it because I already sign my commits but it seems there is a difference between "signing" and "signing off".

I rebased too.
